Opdrachten.js

```diff
@@ -0,0 +1,166 @@
+// Opdracht 1 - Array Methoden
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+
+const tvName = inventory.map((inventoryItem) => {
+    return inventoryItem.name;
+})
+console.log(tvName);
+
+// Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+
+const tvSoldOut = inventory.filter((inventoryItem) => {
+    return inventoryItem.originalStock - inventoryItem.sold === 0;
+})
+console.log(tvSoldOut);
+
+// Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
+
+const hasAmbiLight = inventory.filter((inventoryItems) => {
+    return inventoryItems.options.ambiLight === true;
+})
+console.log(hasAmbiLight);
+
+// Opdracht 1d: Schrijf een functie die alle tv's van laagste naar hoogste prijs sorteert. Log de uitkomst in de console.
+inventory.sort((a, b) => {
+    return a.price - b.price;
+})
+console.log(inventory);
+
+
+//Opdracht 2 - Elementen in de DOM plaatsen
+// Tip: wanneer we meerdere waardes uit een array willen terugbrengen tot één getal of één string, gebruik je hier gewoon een oude vertrouwde for-loop voor!
+// Opdracht 2a: Hoeveel tv's zijn er al verkocht? Schrijf een script dat dit berekent. Log de uitkomst in de console.
+
+let tvsSold = 0;
+
+for (let i = 0; i < inventory.length; i++) {
+    tvsSold += inventory[i].sold;
+}
+
+console.log(tvsSold);
+
+// Opdracht 2b: Zorg ervoor dat dit aantal in het groen wordt weergegeven op de pagina.
+
+const SoldToGreen = document.getElementById("sold-green");
```

Comment:
Variabelnamen moet je in camelCase schrijven
```suggestion
const soldToGreen = document.getElementById("sold-green");
```